commercetools-models/src/test/java/io/sphere/sdk/carts/CartLikeTest.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.DefaultCurrencyUnits.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CartLikeTest {
+
+    private Cart cartWithTaxes;
+    private Cart cartWithoutTaxes;
+
+    @Before
+    public void setUp() throws Exception {
+        cartWithTaxes = SphereJsonUtils.readObjectFromResource("cart-with-taxes.json", Cart.class);
+        cartWithoutTaxes = SphereJsonUtils.readObjectFromResource("cart-without-taxes.json", Cart.class);
+    }
+
```

Comment:
I have minimized the JSON files to only pricing related info. The one without taxes looks quite okay, the other one is... well. Actually in this particular case I find it useful that I'm forced to provide the JSON from real CTP projects directly. It's easy to mistake some calculation or assume a certain price configuration.